cli/common/result.go

```diff
@@ -105,4 +107,6 @@ func init() {
 	resStrings[ResStatusSkipped] = ColorWarn.Sprintf("SKIPPED")
 	resStrings[ResStatusFailed] = ColorErr.Sprintf("FAILED")
 	resStrings[ResStatusExited] = ColorErr.Sprintf("EXITED")
+	resStrings[ResStatusCreated] = ColorYellow.Sprintf("CREATED")
```

Comment:
Why not "ColorGreen"?

Reply:
I was thinking about Ansible logs:

`changed` - yellow (we have two different `changed`: `created` and `updated`)
`ok` - not changed, green